src/FSLibrary/MissionSorobanClassicMixed.fs

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+module MissionSorobanClassicMixed
+
+open StellarCoreSet
+open StellarMissionContext
+open StellarFormation
+open StellarStatefulSets
+open StellarSupercluster
+open StellarCoreHTTP
+
+let sorobanClassicMixed (context: MissionContext) =
+    let coreSet =
+        MakeLiveCoreSet
+            "core"
+            { CoreSetOptions.GetDefault context.image with
+                  invariantChecks = AllInvariantsExceptBucketConsistencyChecks
+                  dumpDatabase = false }
+
+    let context =
+        { context with
+              numAccounts = 2000
```

Comment:
I think it would be nice to have a network setup similar to pubnet in this mission (this network is too tiny to stress test, as it's only a few nodes, and there are no simulated latencies)

Reply:
I thought the main reason we run simulated nodes in the pubnet clusters is because the pod resources are limited, and apply time and DB operations would significantly strain the pods? With soroban TXs being 10-20x more resource intensive than classic ops, this problem would be magnified. If you want to test the network and latencies, would "soroban_upload" not be a better test than invoke host functions?